README.md

```diff
@@ -284,22 +284,23 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Not a Game Studio](http://www.notagamestudio.com/) | Game development studio focused on the experience of the users. | `Lisboa` |
 
 
-## Industry :factory:
+## Health Care :hospital:
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
-| [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://careers.criticalmanufacturing.com/) | Manufacturing business-critical applications. | `Porto` |
-| [FARO Technologies](https://www.faro.com/) [:rocket:](https://www.faro.com/about-faro/career/) | The world’s most trusted source for 3d measurement. | `Porto` |
-| [Malvern Panalytical](https://www.malvernpanalytical.com/) [:rocket:](https://careers.malvernpanalytical.com/careers/?category=24&location=203&search=) | Laboratory analytical instruments design and manufacturing. | `Maia` |
+| [Criteria Marathon](https://www.criteriamarathon.com/) | Software Engineering and Information Technologies in the Medical Sector. | `Porto` |
+| [iLoF](https://ilof.tech/) | Revolutionizing Alzheimer’s drug discovery using bio-photonics and AI. | `Porto` |
+| [Nutrium](https://www.nutrium.com/) [:rocket:](https://nutrium.recruitee.com/) | A nutrition platform that is improving the eating habits of millions of people around the world. | `Braga` `Lisboa` `Remote` |
```

Comment:
Can you short the description a bit?

Currently it's using 96 chars which it's outside the 70 chars maximum [reference](https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md?plain=1#L26)